var/spack/repos/builtin/packages/python/package.py

```diff
@@ -924,15 +962,7 @@ def site_packages_dir(self):
         Returns:
             str: site-packages directory
         """
-        try:
-            return self.config_vars['python_lib']['true']['false']
-        except KeyError:
-            return self.default_site_packages_dir
-
-    @property
-    def default_site_packages_dir(self):
-        python_dir = 'python{0}'.format(self.version.up_to(2))
-        return os.path.join('lib', python_dir, 'site-packages')
+        return self.config_vars['python_lib']['false']['false']
```

Comment:
Note that the default for `site_packages_dir` was previously platlib and is now purelib. This is meant to fix the unit tests (we assume things get installed to `lib/python2.7/site-packages`) but is also probably a better default?

Reply:
Honestly, we might want to consider getting rid of `site_packages_dir`. There is no single `site-packages` directory in Python, different packages may be installed in purelib or platlib.

Reply:
With the latest update, these tests may fail when run with Debian Python. We should really update the tests so that they don't hardcode the directory that things are expected to be installed to.